client/my-sites/plans/current-plan/index.jsx

```diff
@@ -193,7 +195,244 @@ class CurrentPlan extends Component {
 	}
 
 	renderEcommerceTrialPage() {
-		return <div className="current-plan__ecommerce-trial-wrapper">{ this.renderMain() }</div>;
+		const { translate } = this.props;
+
+		return (
+			<>
+				<BodySectionCssClass bodyClass={ [ 'is-trial-plan' ] } />
+
+				<Card className="current-plan__trial-card">
+					<div className="current-plan__trial-card-content">
+						<Badge type="info">{ translate( '5 days left' ) }</Badge>
```

Comment:
I have a few comments here:
1. We have selectors to drive the text here - `getWooExpressTrialDaysLeft()` and `isWooExpressTrialExpired()` were added in #71907.
2. We probably need the `type` and text to be driven by whether the plan has expired.
3. We should use the plural form for the _n days left_ text
```suggestion
		const { hasWooExpressTrialExpired, translate, wooExpressTrialDaysLeft } = this.props; // props are renamed to avoid collision with imported selectors
		
		const badgeType = hasWooExpressTrialExpired ? 'warning' : 'info';
		const badgeText = hasWooExpressTrialExpired
			? translate( 'Your free trial has expired' ) // this could also be 'Your free trial expired on YYYY-MM-DD'
			: translate(
				'%d day left',
				'%d days left',
				{
					count: wooExpressTrialDaysLeft,
					args: [ wooExpressTrialDaysLeft ],
				}
			);

		return (
			<>
				<BodySectionCssClass bodyClass={ [ 'is-trial-plan' ] } />

				<Card className="current-plan__trial-card">
					<div className="current-plan__trial-card-content">
						<Badge type={ badgeType }>{ badgeText }</Badge>
```